dotcom-rendering/src/model/enhanceBlocks.ts

```diff
@@ -35,6 +45,17 @@ class BlockEnhancer {
 		return this;
 	}
 
+	enhanceAdPlaceholders() {
+		if (
+			this.renderingTarget === 'Apps' &&
+			!(this.format.design === 'LiveBlogDesign') &&
+			!(this.format.design === 'DeadBlogDesign')
```

Comment:
> Live/dead blogs have multiple blocks with only one element in so no ads should be added!

I think blog blocks can have multiple elements? See, for example, this block containing a text element followed by a tweet element: https://www.theguardian.com/politics/live/2023/sep/14/starmer-migration-channel-crossings-eu-labour-tackle-people-smuggling-europol-tory-rishi-sunak-humza-yousaf-scotland-china-security-intelligence-uk-politics-latest?page=with:block-6502f2e78f08273df4f007fc#block-6502f2e78f08273df4f007fc

Also, because frontend rewrites CAPI's text blocks to split by paragraph, I think a single text block from CAPI can become multiple text blocks (one per paragraph) by the time it reaches DCR?